packages/backend/src/api/controllers/tvl/DetailedTvlController.ts

```diff
@@ -44,25 +45,25 @@ export class DetailedTvlController {
   async getDetailedTvlApiResponse(): Promise<
     DetailedTvlApiResponse | undefined
   > {
-    const minimumTimestamp = await this.getMinimumTimestamp()
+    const dataTimings = await this.getDataTimings()
```

Comment:
does it really introduce so little overhead to the response time? maybe it would be good to cache it somehow?

Reply:
It is acceptable - actually lower than before.

#### Timing overhead distribution:
- Current (production replica synced up to yesterday): 
  - `findCountsForHash: 19.291166067123413ms`
  - `database query`
![image](https://github.com/l2beat/l2beat/assets/67391475/3b16ed26-66b7-4418-bc40-63c6c6c32383)

- Previous method:
  - `getMinimumTimestamp: 47.10783314704895ms`
  - `database query` - skipped since we have quite a few queries there